src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/config/SecretConfigurations.kt

```diff
@@ -3,13 +3,14 @@ package it.pagopa.ecommerce.eventdispatcher.config
 import it.pagopa.ecommerce.commons.utils.ConfidentialDataManager
 import it.pagopa.generated.pdv.v1.ApiClient
 import it.pagopa.generated.pdv.v1.api.TokenApi
+import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
 @Configuration
 class SecretConfigurations {
-  @Bean
+  @Bean("personalDataVaultApi")
```

Comment:
We could call this `eCommercePersonalDataVaultApi` for consistency, what do you think?

```suggestion
  @Bean("eCommercePersonalDataVaultApi")
```

Reply:
I agree, I'm going to rename it